server.js

```diff
@@ -0,0 +1,40 @@
+const teamToIDs = {
+    "lakers": "1610612747",
+    "warriors": "1610612744",
+    "heat": "1610612748",
+    "suns": "1610612756"
+}
+const express = require('express')
+const app = express()
+const path = require('path')
+const urllib = require('urllib')
+const body = require('body-parser')
+const { json } = require('body-parser')
+app.use(express.static(path.join(__dirname, 'dist')))
+app.use(express.static(path.join(__dirname, 'node_modules')))
+
+app.get('/team/:teamName', function (req, res) {
+    const team = req.params.teamName
```

Comment:
you can deconstruct objects
`const { teamName } = req.params`

is the same as
`const teamName = req.params.teamName`